cmd/vclusterctl/cmd/create.go

```diff
@@ -77,6 +78,16 @@ func (cmd *CreateCmd) Run(ctx context.Context, args []string) error {
 		config.PrintDriverInfo("create", driver, cmd.log)
 	}
 
+	vclusters, err := find.ListVClusters(ctx, "", "", cmd.Namespace, cmd.log)
+	if err != nil {
+		return err
+	}
+	for _, v := range vclusters {
+		if v.Namespace == cmd.Namespace && v.Name != args[0] {
+			return fmt.Errorf("there is already a virtual cluster in namespace %s", cmd.Namespace)
+		}
+	}
```

Comment:
I think this should go into the helm driver only, as there can be many vClusters in the platform in different host clusters

Reply:
I had it there before, but do we support creating multiple virtual clusters in the same ns with the platform?

Reply:
it seems that the `ListOSS` function should work with platform created virtual clusters too, since it's fetching the statefulset/deployment